book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
 ----
 
 [[_undoing_merges]]
-==== Undoing Merges
+==== Pag-undo sa mga merge
```

Comment:
change capitalization `Pag-undo sa mga merge` to `Pag-undo sa mga Merge`

Reply:
done change